settings.gradle

```diff
@@ -19,16 +19,9 @@ if (localProperties != null) {
     if (appServicesLocalPath != null) {
         logger.lifecycle("Local configuration: substituting application-services modules from path: $appServicesLocalPath")
 
-        includeBuild(appServicesLocalPath) {
-            dependencySubstitution {
-                substitute module('org.mozilla.appservices:fxaclient') with project(':fxa-client-library')
-                substitute module('org.mozilla.appservices:logins') with project(':logins-library')
-                substitute module('org.mozilla.appservices:places') with project(':places-library')
-                substitute module('org.mozilla.appservices:rustlog') with project(':rustlog-library')
-            }
-        }
+        includeBuild(appServicesLocalPath)
 
     } else {
-        logger.lifecycle("Local configuration: application-services substitution path missing. Specify it via '$settingAppServicesPath' setting.")
+        logger.lifecycle("Local configuration: application-services substitution path missing. You may specify it via '$settingAppServicesPath' setting.")
```

Comment:
There are good reasons not to specify it so I wanted to change the wording of these to be less imperative.